app/views/contacts/index.html.haml

```diff
@@ -22,9 +22,10 @@
           .no_contacts
             %h3
               = t('.no_contacts')
-            %p
-              != t('.no_contacts_message',
-                   :community_spotlight => link_to(t('.community_spotlight'), community_spotlight_path))
+            - if AppConfig.settings.community_spotlight.enable?
+              %p
+                != t('.no_contacts_message',
```

Comment:
- Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
- Use the new Ruby 1.9 hash syntax.